README.md

```diff
@@ -12,6 +12,153 @@
+## Table of Contents
 
-Declarative library for composing spectrum visualizations in React.
+- [React Spectrum Charts](#react-spectrum-charts)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+    - [Key Features:](#key-features)
+  - [Installation](#installation)
+      - [npm](#npm)
+      - [yarn](#yarn)
+  - [Usage](#usage)
+    - [Example](#example)
+  - [Spectrum (Adobe Design System) Integration](#spectrum-adobe-design-system-integration)
+  - [API](#api)
+  - [Storybook](#storybook)
+  - [Contributing](#contributing)
+  - [License](#license)
+    - [Apache License 2.0 Summary](#apache-license-20-summary)
+  - [Roadmap](#roadmap)
+
+## Overview
+
+`react-spectrum-charts` is a declarative charting library for composing charts in React. It empowers you to effortlessly create visually stunning charts following the Adobe design system ([spectrum](https://spectrum.adobe.com)) with minimal code. Understanding of the grammar of graphics or lower-level libraries like D3 or Vega is not needed.
+
+### Key Features:
+
+-   **Responsive Charts:** `react-spectrum-charts` ensures that your charts adapt beautifully to different screen sizes and orientations, providing a seamless user experience on various devices.
+
+-   **Interactive Charts:** Create engaging charts that respond to user interactions, such as hovering, clicking, and more. Enhance data exploration and understanding with interactive elements.
+
+-   **Supported Chart Types:**
+
+    -   **Area**
+    -   **Bar**
+    -   **Line**
+
+-   **Supplemental Components:**
+
+    -   **Annotation:** Add callouts to your charts.
+    -   **Popover:** Add actions and additional content to selected data points.
+    -   **Tooltip:** Add tooltip content to hovered points.
+    -   **Trendline:** Add regressions or moving averages to your charts.
+
+-   **Modular Components:**
+
+    -   **Axis**
+    -   **Legend**
+    -   **Title**
+
+-   **Simple and Intuitive API:** We've designed an API that's easy to understand and work with, allowing you to create beautiful charts quickly. No need to delve into the complexities of the grammar of graphics or wrestle with low-level libraries.
+
+-   **Built on Vega:** `react-spectrum-charts` is built on top of Vega, a robust and versatile visualization grammar for expressive, concise, and interactive data visualizations. This foundation ensures both flexibility and performance.
+
+`react-spectrum-charts` simplifies the charting process, making it accessible to a broader audience, including developers who may not have prior experience in data visualization. With just a few lines of code, you can create charts that meet the Adobe design system's standards, enriching your React applications with informative and visually appealing data representations.
+
+## Installation
+
+-   Instructions for installing your library.
+-   Include code snippets for installation via npm or yarn.
+-   Mention any peer dependencies.
```

Comment:
I'm assuming these were left in on accident.

Reply:
Yeah...